### PR TITLE
Add notification action override docs for Android

### PR DESCRIPTION
**Pull request by jugarrit**
@dannytranlx 
@jpjoyal 
@mspensieri 
@alavers

### CODE REVIEW

File: source/includes/customization/_notification_action.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+## Notification Action Intent Override
+
+The default behaviour of tapping on a push or in-app notification is to open the Smooch `ConversationActivity` intent.
```
Comment (dannytranlx) [inline source/includes/customization/_notification_action.md]: > behavior

-or or -our?

File: source/includes/customization/_notification_action.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+If you want to change this behaviour, simply override the following resource in `settings.xml` to the value of your choice:
+
+```xml
+<resources>
+  <string name="Smooch_settings_notificationIntent">io.smooch.ui.ConversationActivity</string>
+</resources>
+```
+
+Note that the specified intent must extend from the `Activity` class.
```
Comment (dannytranlx) [inline source/includes/customization/_notification_action.md]: Wrap around an `<aside>` maybe?

**Comment by dannytranlx** [comment]
👍

File: source/includes/customization/_notification_action.md (hunk 1 above)
Comment (jugarrit) [inline source/includes/customization/_notification_action.md]: One is American, one is English/Canadian.
/shrug

File: source/includes/customization/_notification_action.md (hunk 2 above)
Comment (jugarrit) [inline source/includes/customization/_notification_action.md]: Sure

**Pull request merged**